client/index.js

```diff
@@ -10,23 +10,25 @@ initialDoc: 'https://pdftron.s3.amazonaws.com/downloads/pl/webviewer-demo.pdf',
   var annotManager = docViewer.getAnnotationManager();
 
   // Save when annotation change event is triggered (adding, modifying or deleting of annotations)
-  annotManager.on('annotationChanged', function(e, annots) {
+  annotManager.on('annotationChanged', function(annots, action, options) {
     // If the event is triggered by importing then it can be ignored
     // This will happen when importing the initial annotations from the server or individual changes from other users
-    if (e.imported) return;
+    if (options.imported) return;
 
-    const xfdfStrings = annotManager.getAnnotCommand();
-    annots.forEach(function(annot) {
-      savexfdfString(DOCUMENT_ID, annot.Id, xfdfStrings);
+    annotManager.getAnnotCommand().then(function (xfdfStrings) {
+      annots.forEach(function(annot) {
+        savexfdfString(DOCUMENT_ID, annot.Id, xfdfStrings);
+      });
     });
   });
 
   // Load annotations when document is loaded
   docViewer.on('documentLoaded', function() {
     loadxfdfStrings(DOCUMENT_ID).then(function(rows) {
       JSON.parse(rows).forEach(col => {
-        const annotations = annotManager.importAnnotCommand(col.xfdfString);
-        annotManager.drawAnnotationsFromList(annotations);
+          annotManager.importAnnotCommand(col.xfdfString).then(function(annotations) {
+          annotManager.drawAnnotationsFromList(annotations);
+		    });
```

Comment:
this one looks weird. here is what it looks like in VS code:
![image](https://user-images.githubusercontent.com/54084743/71287999-adcb6300-231e-11ea-8b4e-492bf68b8ebb.png)